main/res/values/strings.xml

```diff
@@ -1248,6 +1248,7 @@
     <string name="waypoints_tabtitle" tools:ignore="PluralsCandidate">Waypoints (%d)</string>
     <string name="cache_waypoints_add">Add Waypoint</string>
     <string name="cache_waypoints_add_currentLocation">Add current location</string>
+    <string name="cache_waypoints_hide_visited_waypoints">Hide visited Waypoints</string>
```

Comment:
"Waypoints" should be lower-case "waypoints" as it is in the middle of a sentence